src/main/scala/sbtcatalysts/CatalystsPlugin.scala

```diff
@@ -398,7 +398,8 @@ trait CatalystsBase {
     libraryDependencies ++= Seq(
       "com.github.mpilquist" %%% "simulacrum" % v.vers("simulacrum") % "compile-time"),
      ivyConfigurations += config("compile-time").hide,
-     unmanagedClasspath in Compile ++= update.value.select(configurationFilter("compile-time"))
+     unmanagedClasspath in Compile ++= update.value.select(configurationFilter("compile-time")),
+     addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full)
```

Comment:
There is a 2.1.1.  Is there any reason not to use that?

Reply:
Ah didn't see it.